flowzone.yml

```diff
@@ -1351,13 +1376,19 @@ jobs:
 
       # update the existing branch(head) name reference(pointer) to the new commit object sha
       - <<: *updateGitReference
+        if: |
+          github.event.pull_request.merged &&
```

Comment:
Why was this not needed before? Was it a bug?

Reply:
It used to be on line 93, as part of the anchor. It makes more sense to have the condition where it is used.